app/service/rpc_service.ts

```diff
@@ -186,18 +186,46 @@ class RpcService {
         init.body = lengthPrefixMessage(requestData);
 
         const reader = (await this.fetch(url, "stream", init))?.getReader();
+        let buffer = new Uint8Array();
+        let bufferedLength = 0;
+        let totalLength = 0;
```

Comment:
uber nit: I got `buffered` and `total` confused while reading this, maybe `messageExpectedLength` might be more clear?

Reply:
Done